App/src/pages/RemoveUserFromGroup.tsx

```diff
@@ -0,0 +1,124 @@
+            const _state = location.state as { gid: string; name: string };
+            setLocationState(_state);
+            console.log(locationState.gid);
+            getMyGroup(locationState.gid);
+        }
+    }, [location]);
+    const userList = myGroup.members;
+    const [filteredList, setFilteredList] = useState(userList);
+    const filterBySearch = (event: { target: { value: string } }) => {
+        const query = event.target.value;
+
+        let updatedList = [...userList];
+
+        updatedList = updatedList.filter((item) => {
+            return item.email.toLowerCase().indexOf(query.toLowerCase()) !== -1;
+        });
+
+        setFilteredList(updatedList);
+    };
+
+    function removingUser(uid: string) {
+        RemoveUserFromGroup(myGroup.id, uid)
+            .then(
+                () => {
+                    console.log("User Added");
+                },
+                (rej) => {
+                    console.log(rej);
+                }
+            )
+            .catch((err) => {
+                console.log(err);
+            });
+    }
+
+    return (
+        <Container maxW="lg" py={{ base: "12", md: "24" }} px={{ base: "0", sm: "8" }}>
+            <Stack spacing="8">
+                <Box
+                    py={{ base: "0", sm: "8" }}
+                    px={{ base: "4", sm: "10" }}
+                    bg={{ base: "transparent", sm: "bg.surface" }}
+                    boxShadow={{ base: "none", sm: "2xl" }}
+                    borderRadius={{ base: "none", sm: "xl" }}
+                >
+                    <Stack spacing="6">
+                        <img src={logo} alt="Logo" width="50px" height="50px" style={{ margin: "auto" }} />
+                        <Heading textAlign="center" size="lg">
+                            Remove User from Group: {locationState.name}
+                        </Heading>
+
+                        <FormControl>
+                            <FormLabel htmlFor="search-text">Search:</FormLabel>
+                            <Input id="groupName" type="text" onChange={filterBySearch} />
+                        </FormControl>
+
+                        <div id="item-list">
+                            <ol>
+                                {filteredList.map((item, index) => (
+                                    <HStack justify="space-between" key={index}>
+                                        <li>{item.email}</li>
+                                        <Button
+                                            style="primary"
+                                            variant="outline"
+                                            width="22%"
+                                            height="40px"
+                                            justifyContent={"flex-end"}
+                                            onClick={() => removingUser(item.id)}
+                                        >
+                                            Select
```

Comment:
Can you change the button content to "Remove"?

Reply:
done